pkg/cfaws/profiles.go

```diff
@@ -8,10 +8,12 @@ import (
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/bigkevmcd/go-configparser"
+	"github.com/common-fate/granted/pkg/browsers"
 	"github.com/fatih/color"
 )
 
 type CFSharedConfig struct {
+	Opts browsers.BrowserOpts
```

Comment:
My git feel is that this doesn't belong on the sharedconfig struct, as it is not specific to each profile.
My preference is to pass the duration into the assumers, make a dedicated Opts struct for the assumers.
And another for the Browser. this keeps our packages from having too much coupling